scripts/combine/setupCombineWMass.py

```diff
@@ -66,6 +65,8 @@ def main(args,xnorm=False):
     wmass = datagroups.wmass
     wlike = datagroups.wlike
 
+    constrainMass = False
```

Comment:
Hmm previously we constrained the mass also for the unfolding, not sure if we should or not